src/redesign/components/Donate.jsx

```diff
@@ -16,8 +17,12 @@ export default function Donate() {
     <div>
       <Header />
       <PageHeader title="Donate" backgroundColor={style.colors.BLUE_98}>
-        <p style={{margin: 0}}>PolicyEngine is a nonprofit fiscally sponsored by the PSL Foundation in the United States.</p>
+        <p style={{margin: 0, fontFamily: style.fonts.BODY_FONT}}>Explore Brookings&apos; research and commentary to deepen your 
```

Comment:
Could we revert back to the non-placeholder text? Thanks!

Reply:
Sure, I've reverted it.